src/config_magic.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 #define MAGIC_ITEMS_MAX        64
 #define SPELL_MAX_LEVEL         8
 #define MAGIC_OVERCHARGE_LEVELS (SPELL_MAX_LEVEL+1)
-#define MAGIC_TYPES_COUNT      30
+#define MAGIC_TYPES_COUNT      31
```

Comment:
Remove it instead. The only other place this is used you can replace it with `magic_conf.spell_types_count`